src/index.ts

```diff
@@ -172,15 +154,47 @@ async function initWatch(mongoData: MongoDataType, rabbitData: RabbitDataType, o
     console.log(`MTR: ===> successful connection to collection: ${mongoData.collectionName}`);
 }
 
-export function sendMsg(queue: QueueObjectType, msg: any) {    
+
+/**
+ * formatMsg - parse msg and send it to queue
+ * @param {QueueObjectType}             queue     - queue object 
+ * @param {MTROptions}                  options   - options for package
+ * @param {mongodb.ChangeEvent<Object>} event     - mongo change event
+ * @param {MongoDataType}               mongoData - mongoData options
+ */
+function formatMsg(
+    queue: QueueObjectType,
+    options: MTROptions,
+    event: mongodb.ChangeEvent<Object>,
+    mongoData: MongoDataType) {
+    const formattedData = options.prettify ? 
+        queue.middleware == undefined? 
+            defaultMiddleware(prettifyData(event, options), mongoData.collectionName): 
+            queue.middleware(prettifyData(event, options), mongoData.collectionName)
+        : event;
+    
+    if (formattedData !== null && formattedData !== undefined) {
+        (Array.isArray(formattedData))? 
+            formattedData.forEach(dataContent => sendMsg(queue, dataContent)): 
+            sendMsg(queue, formattedData);
+    }
+}
+/**
+ * sendMsg function to queue - by exchange or direct queue
+ * @param {QueueObjectType} queue   - queue object 
+ * @param {any}             msg     - formatted msg
+ */
+export function sendMsg(queue: QueueObjectType, msg: any) {   
+    console.log('sending msg'); 
```

Comment:
not informative. remove it or use the log and print a more informative data in the format

Reply:
done